cmd/lnmuxd/run.go

```diff
@@ -397,3 +364,40 @@ func initDistributedLock(cfg *DistributedLockConfig) (func(), error) {
 		Logger:        log,
 	})
 }
+
+func initInstrumentationServer(instAddress string) *http.Server {
```

Comment:
For review, it would be useful to have this method extraction in a separate preparatory commit.